Gems/PhysX/Code/Source/HingeJointComponent.h

```diff
@@ -24,9 +26,24 @@ namespace PhysX
         HingeJointComponent(
             const JointComponentConfiguration& configuration, 
             const JointGenericProperties& genericProperties,
-            const JointLimitProperties& limitProperties);
+            const JointLimitProperties& limitProperties,
+            const JointMotorProperties& motorProperties = JointMotorProperties());
         ~HingeJointComponent() = default;
 
+        // JointInterfaceRequestBus::Handler
+        float GetPosition() override;
+        float GetVelocity() override;
+        AZ::Transform GetTransform() override;
+
+        // JointMotorRequestBus ::Handler
```

Comment:
update: Is this correct? It seems like this should be grouped with the member functions from `JointInterfaceRequestBus`?